source/manual/dns.html.md

```diff
@@ -31,80 +27,34 @@ We use a few domains:
   We maintain records which point to Bouncer so that these URLs redirect.
 - `publishing.service.gov.uk` and `govuk.service.gov.uk` are where GOV.UK lives.
 
-## Making changes to publishing.service.gov.uk
-
-To make a change to this zone, begin by adding the records to the yaml file for
-the zone held in the [DNS config repo](https://github.com/alphagov/govuk-dns-config).
-
-We use a Jenkins job that publishes changes to `publishing.service.gov.uk`. The
-job uses [Terraform](https://www.terraform.io/) and pushes changes to the
-selected provider.
-
-### Deployment
-
-**Always `plan` first, check that the output is what you expect, then `apply`.**
-
-When the changes have been reviewed and merged, you can deploy them in your terminal
-by using at least version `v2.15.0` of [`gds-cli`][gds-cli].
+## DNS for `*.service.gov.uk` domains
 
-**You MUST deploy to BOTH DNS providers. We use both for redundancy.**
+GOV.UK Technical 2nd Line are responsible for delegating DNS to other government services.
+Note that we __do not__ manage any other DNS records: if you get a request concerning anything other than `NS` records, it should be rejected. See the [SRE interruptible documentation](https://docs.google.com/document/d/1QzxwlN9-HoewVlyrOhFRZYc1S0zX-pd97igY8__ZLAo/edit#heading=h.wg0s4ugkpdpc) for details.
 
-You should **always**:
+When you've verified the authenticity of the request as per the SRE docs above, you should:
 
-1. `plan` in each DNS provider and check the console output is what you expect.
-2. `apply` in each DNS provider until you see that your changes has been applied.
-    There are [circumstances where terraform is not able to apply the changes](#google-based-caveats)
-    in 1 run and need **multiple** runs.
+1. Make the changes in govuk-dns-config (see [example](https://github.com/alphagov/govuk-dns-config/pull/851))
+1. Deploy the changes via the [Deploy_DNS Jenkins job](https://deploy.blue.production.govuk.digital/job/Deploy_DNS/). You'll need production admin access to do this.
 
-Deployment is done by first obtaining (first time set up) your GitHub credentials
-by creating a read-only GitHub personal access token. This [GitHub personal access token](https://github.com/settings/tokens) should be created with the `read:org`
-scope only.
+Deploying is a multi-step process:
```

Comment:
Content looks good to me, I think we just need to bump up the list numbers however, all steps are currently showing as "1".

```markdown
1. Run a `plan` of the deployment, against the `aws` provider.
   - You can get the necessary AWS credentials by running `gds aws govuk-production-admin -e`.
2. Check that the output is what you expect.
3. Retrieve a Google OAuth access token.
   - You'll need the gcloud CLI (`brew install --cask google-cloud-sdk`).
   - Then run the following - you'll be prompted to login to your Google account to allow Google Cloud SDK access your Google Account. The token will be printed in the terminal:
   - `gcloud config set project govuk-production; gcloud auth login --brief; gcloud auth print-access-token`
   - Use this token for the `GOOGLE_OAUTH_ACCESS_TOKEN` field in the next step.
4. Now run a `plan` of the deployment against the `gcp` provider.
   - You'll still need to provide all the AWS credentials as per step 1. This is because the Terraform state is held in an S3 bucket.
5. Check that the output is what you expect.
   - It's normal to see changes in TXT records relating to escaping of quotes. You can safely ignore these if they don't change any of the content of the record. This is a bug in the way we handle splitting long TXT records between AWS and GCP in our [YAML -> Ruby -> Terraform process](https://github.com/alphagov/govuk-dns).
6. Finally, run an `apply` deployment for both `aws` AND `gcp`. (The order doesn't matter).
   - Sometimes, the GCP deployment requires multiple runs. This is because, in order to change a DNS record, the Google provider deletes and re-adds that record. This can cause a [race condition](https://github.com/alphagov/govuk-dns/issues/67) where Google tries to create the new one before it has successfully deleted the old one. In this case, the build will fail, and you just need to re-run the GCP `apply` job.
```

Reply:
Thanks Martin! That's deliberate, as GitHub's markdown parser will automatically increment the numbers (saves you having to rewrite all the numbers if you reorder or remove any of them)

Reply:
Thanks very much, I didn't know that 👍